L/LVGL_SDL/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "LVGL_SDL"
+version = v"9.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/lvgl/lvgl.git", "4f086111a107718b719a45e2cc422f5b756d7edd"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/lvgl
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/*.patch
+if [[ "${target}" == "x86_64-apple-darwin14" ]]
+then
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/x86_64-apple-darwin14/*.patch
+fi
+cp lv_conf_template.h lv_conf.h
+cp lv_conf.h ..
+
+cmake -B build \
+      -DBUILD_SHARED_LIBS=ON \
+      -DLIB_INSTALL_DIR=${prefix}/lib \
+      -DLV_CONF_BUILD_DISABLE_DEMOS=1 \
+      -DLV_CONF_BUILD_DISABLE_EXAMPLES=1 \
+      -DLV_CONF_INCLUDE_SIMPLE=OFF \
+      -DCMAKE_INSTALL_PREFIX=${prefix} \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release
+cmake --build build --parallel ${nproc}
+cmake --install build
+"""
+
```

Comment:
Step 2 for using `require_macos_sdk`:
```suggestion

sources, script = require_macos_sdk("11.1", sources, script)

```
Here 11.1 is just an example. You can also use something newer (ideally the older the better because then more people can use it, but of course it must be new enough for the package to build. Look at the file `platforms/macos_sdks.jl` to see which SDKs we currently have there; more can be added as needed)